tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -1350,6 +1361,15 @@ final class SomeClass {}',
 final class SomeClass {}',
         ];
 
+        yield 'PHPDoc with generics must not crash' => [
+            '<?php
+
+/**
+ * @param \Iterator<mixed, \SplFileInfo> $iter
+ */
+function foo($iter) {}',
+        ];
+
```

Comment:
side note:
with recently introduced "test class is contract", while adding support for generics, you would need to put it behind rule option, as otherwise it would change contract behaviour.